### DISCUSSION TITLE
Error Codes

### DISCUSSION THREAD

**Discussion by nyxb** [microsoft/pylance-release#4487] [General]
There is a systematic way to retrieve the error codes and messages used by Pylance.

**Comment by rchiodo** [comment]
Can you describe your request in more detail? Why do you need the error messages used by Pylance? (We have no 'codes' at the moment).

**Comment by rchiodo** [comment]
Most of the error messages in Pylance come from here:
https://github.com/microsoft/pyright/blob/main/packages/pyright-internal/src/localization/package.nls.en-us.json

But that is about to change as we're soon going to support multiple languages.
